e2e/kube.go

```diff
@@ -130,7 +127,26 @@ spec:
           privileged: true
           capabilities:
             add: ["SYS_PTRACE", "SYS_RAWIO"]
-`
+`, deploymentName, isHostNetwork, targetNodeLabel)
+}
+
+func createDebugDeployment(ctx context.Context, kube *Kubeclient, manifest string) error {
```

Comment:
okay we should align on naming, either put back as daemonset if we're gonna change it to a daemonset in reality or renaming everything else to be deployment

Reply:
Fair, i can change the naming to daemonsets. Did that initially but then decided not to for some reason